radis/spectrum/compare.py

```diff
@@ -319,116 +318,54 @@ def get_residual(
         how to normalize. ``'max'`` is the default but may not be suited for very
         noisy experimental spectra. ``'area'`` will normalize the integral to 1.
         ``'mean'`` will normalize by the mean amplitude value
-

```

Comment:
is it normal to have removed this ?

Reply:
That was a mistake. I put back the "See Also"

Do you really want to have something else than the L2-norm implemented ?

Reply:
Ok, it may be overcomplicating things.